docs/reference/contributing/guidelines/workflow.md

```diff
@@ -80,14 +102,28 @@ Each pull request goes through the following workflow:
 
 Labels that the Mbed OS maintainers add to a pull request represent the pull request workflow states. The Mbed OS maintainers are responsible for moving pull requests through the workflow states.
 
+Each state is time boxed. In most cases, this should be sufficient time to move to an another state.
+
 ##### Reviews
 
-All pull requests must be reviewed. The Mbed OS maintainers determine the most suitable person to review the pull request and tag that person accordingly. If a pull request requires a review from partners, the maintainers tag the corresponding GitHub group instead.
+All pull requests must be reviewed. The mbed CI bot determine the most suitable person to review the pull request and tag that person accordingly.
+
+Time: 3 days
```

Comment:
Better to explain this specifically. E.g.
Is 3 days the maximum allowance for the initial review to take place, the full review cycle or for each review iteration ? We need to clarify this.